.github/dependabot.yml

```diff
@@ -27,6 +27,9 @@ updates:
       day: "sunday"
     ignore:
       - dependency-name: "Moq"
+    labels:
+      - vs-bicep-dependencies
```

Comment:
Where does this get used?

Reply:
Not currently used, just seemed like an easy way for me to find all dependabot issues related to Bicep in VS when I want to.